2019Labs/CustomSecurityContent/documentation/lab2_openscap.adoc

```diff
@@ -167,155 +185,59 @@ image::scap_workbench_tailoring.png[SCAP Workbench content customization]
 
 . Now let's run a test scan with the new custom profile we just created.
 Click *Scan* and inspect the results.
-When prompted for the password for *lab-user*, type *r3dh4t1!*.
+When prompted for the password for Lab User, type *r3dh4t1!*.
 This will take a few minutes so feel free to move on with the lab exercise and not wait until the scan is completed.
+Close the *Diagnostics* window.
 +
 image:lab1.2-scapworkbenchscan.png[500,500]
 
-. (Optional) You can save it to a tailoring file by selecting File->Save Customization Only.
+. (Optional) You can save the customization to a tailoring file by selecting File->Save Customization Only.
 +
 image:lab1.2-savecustomization.png[300,300]
 
-== Lab 1.3 Security Remediations with OpenSCAP and Ansible
+== Security Remediations with OpenSCAP and Ansible
 Putting the machine into compliance (for example by changing its configuration) is called *remediation* in the SCAP terminology.
 Remediation changes the configuration of the machine and it is possible that you will lock yourself out or disable important workloads!
 As a result, it is best practice to test the remediation changes before deploying.
 
 . All remediations will be executed locally.
 
-. Let's generate an Ansible playbook that will put the *openscap.example.com* machine into compliance.
-Let's go ahead and generate a playbook from the results:
+. Let's generate an Ansible playbook that will put your machine into compliance.
+We will generate a playbook from the scan results:
 +
 Use the `--fix-type ansible` option to request an ansible playbook with the fixes:
 +
 ----
-oscap xccdf generate fix --fix-type ansible --result-id "" arf.xml > playbook.yml
+oscap xccdf generate fix --fix-type ansible --result-id "" /tmp/arf.xml > playbook.yml
 ----
-
-. (Optional) Generate bash remediation script and run it on target machine(s). This can be accomplished by running:
-* use `--fix-type bash` to request a bash script with the fixes
-* after the script is generated change its permissions so that we can run it
 +
-----
-oscap xccdf generate fix --fix-type bash --result-id "" arf.xml > bash-fix.sh
-chmod +x bash-fix.sh
-./bash-fix.sh
-----
-
-. Notice that in both cases we are using empty result-id.
-This is a trick to avoid specifying the full result ID.
-
-. We will focus on the Ansible remediation options in this next part of the lab exercise.
-
-. Let's open the generated playbook using a text editor.
-In this example, we will use `gedit` as our text editor (but feel free to use `vi` as well).
+Check the output using a text editor:
 +
 ----
 gedit playbook.yml
 ----
 +
-The file contents look like this:
+Keep in mind that even if the comments in the headers say that it is an Ansible Role, it is not an
```

Comment:
We don't have to advertise this playbook/role/remediation confusion - I would perhaps omit this, or state that it is a playbook that can be converted to an Ansible role.
The comment won't be there when the scanner is updated, and the audience may not know what is the difference between a playbook and a role.